### DISCUSSION TITLE
About teams

### DISCUSSION THREAD

**Discussion by tszyuloveyou** [spatie/laravel-permission#2624] [General]
If I have many committees(team), eg: Internal Publications Committee, Event Organizing Committee, Branding and Communication Committee.
if user have multiple committees and I want that user need not to change current team and need not to set current team id, just check if he has permission or he has any roles that have permission or he has any committees roles that have permission,
May I know that how I can do.

**Comment by drbyte** [comment]
Usually for that situation I would simply treat each committee as a role, not as a "team".
